staging/src/k8s.io/kubectl/pkg/cmd/testing/fake.go

```diff
@@ -786,6 +786,7 @@ func testDynamicResources() []*restmapper.APIGroupResources {
 			VersionedResources: map[string][]metav1.APIResource{
 				"v1": {
 					{Name: "bars", Namespaced: true, Kind: "Bar"},
+					{Name: "applysets", Namespaced: false, Kind: "ApplySet"},
```

Comment:
Without this, I get a failure earlier than the real `kubectl` binary does:

```
error: invalid parent reference "applysets.company.com/my-set": no matches for company.com/, Resource=applysets
```
Coming from `ParseApplySetParentRef`, because the rest mapping doesn't have this resource. Seemingly real kubectl does (presumably from discovery).

Reply:
Yes, I think "no matches for ..." is an artifact of the test factory mocks.